test/administrative-sdkSpec.js

```diff
@@ -65,7 +65,7 @@ describe('Administrative SDK', () => {
     sdk.getPronunciationChallenge(1, 2);
     sdk.listPronunciationChallenges(1);
     sdk.deletePronunciationChallenge(1);
-    sdk.createSpeechChallenge(1);
+    sdk.createSpeechChallenge(1, 2);
```

Comment:
I guess it makes more sense to not pass int 2 but 'audioBlob' or some other more descriptive dummy object()?